### PR TITLE
Added Zenith chain mainnet and testnet

### PR DESCRIPTION
**Pull request by ammar-lakho**

### CODE REVIEW

File: _data/chains/eip155-79.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "name": "ZENITH",
+      "symbol": "ZENITH",
+      "decimals": 18
+    },
+    "infoURL": "https://www.zenithchain.co/",
+    "chainId": 79,
+    "networkId": 79,
+    "explorers": [{
+      "name": "zenith scan",
+      "url": "https://scan.zenithchain.co/",
```
Comment (ligi) [inline _data/chains/eip155-79.json:19]:
```suggestion
      "url": "https://scan.zenithchain.co",
```

File: _data/chains/eip155-81.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "name": "test ZENITH",
+      "symbol": "tZENITH",
+      "decimals": 18
+    },
+    "infoURL": "https://www.zenithchain.co/",
+    "chainId": 81,
+    "networkId": 81,
+    "explorers": [{
+      "name": "villinius zenith scan",
+      "url": "https://vilinius.scan.zenithchain.co/",
```
Comment (ligi) [inline _data/chains/eip155-81.json:18]:
```suggestion
      "url": "https://vilinius.scan.zenithchain.co",
```

**Review by ligi — CHANGES_REQUESTED** [review]
The field `shortName` is missing

**Comment by ammar-lakho** [comment]
> The field `shortName` is missing

Added the shortName field.

**Comment by ammar-lakho** [comment]
@ligi How long after merging will the network start showing up on https://chainlist.org/ ?

**Pull request merged**